### PR TITLE
Rox 11211 collector ci migration images

### PR DESCRIPTION
**Pull request by JoukoVirtanen**
Migrating images job to openshift ci.

### CODE REVIEW

File: ci-operator/config/stackrox/collector/stackrox-collector-ROX-11211-collector-ci-migration-images.yaml [hunk 1]
```diff
@@ -13,6 +16,27 @@ images:
...
+  dockerfile_path: collector/container/Dockerfile.gen
+  from: ubi-minimal
+  context_dir: collector/container/rhel
+  inputs:
+    collector-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/collector/.
+  to: collector-full
+promotion:
```
Comment (erthalion) [inline ci-operator/config/stackrox/collector/stackrox-collector-ROX-11211-collector-ci-migration-images.yaml:37]: What exactly is this for? I was thinking that image will be published via a separate test step?

**Review by robbycochran — APPROVED** [review]

**Pull request merged**